agent/handlers/v1/credentials_handler.go

```diff
@@ -85,7 +85,7 @@ func CredentialsHandlerImpl(w http.ResponseWriter, r *http.Request, auditLogger
 func processCredentialsRequest(credentialsManager credentials.Manager, r *http.Request, credentialsID string, errPrefix string) ([]byte, string, string, *handlersutils.ErrorMessage, error) {
 	if credentialsID == "" {
 		errText := errPrefix + "No Credential ID in the request"
-		seelog.Infof("%s. Request IP Address: %s", errText, r.RemoteAddr)
+		seelog.Errorf("Error processing credential request, requestIPAddress=%s credentialsID=%s: %s", r.RemoteAddr, credentialsID, errText)
```

Comment:
might be better not logging the credentialsID from a security perspective.

Reply:
I have refactored it to not print the credentials ID, and instead when available it will print the task ARN